### ISSUE TITLE
🐛 continuous ejecting after ripping has completed

### ISSUE THREAD

**Issue by Tjkent88** [automatic-ripping-machine/automatic-ripping-machine#820]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### I have checked the wiki

- [X] I have searched the wiki for this problem

### Does this issue exist in the latest version?

- [X] I'm using the latest release

### Describe the bug?

Once ripping is complete, the drive is ejected, but the drive continues to eject until I restart the container. I know that the drive works, because I used to have this running with the ubuntu script. 

### To Reproduce

I used the docker-container script, and run using docker-compose.

I am using the container inside a proxmox vm. 

```
arm-rippers:
    image: automaticrippingmachine/automatic-ripping-machine:nvidia
    container_name: arm-rippers
    restart: unless-stopped
    privileged: true
    runtime: nvidia
    deploy:
      resources:
        reservations:
          devices:
            - capabilities: [gpu]
    devices:
      - /dev/sr0:/dev/sr0
      - /dev/sr1:/dev/sr1
      - /dev/sr2:/dev/sr2
      - NVIDIA_VISIBLE_DEVICES=all
      - NVIDIA_DRIVER_CAPABILITIES=all
    environment:
      - ARM_UID=1002
      - ARM_GID=1002
    volumes:
      - /home/tkent/appdata/arm:/home/arm
      - /mnt/nfs/main/media/arm:/home/arm/Music
      - /home/tkent/appdata/arm/logs:/home/arm/logs
      - /mnt/nfs/main/media/arm:/home/arm/media
      - /home/tkent/appdata/arm/config:/etc/arm/config
      - /mnt/nfs/main/media/arm/transcode:/home/arm/transcode
      - /dev/shm:/home/arm/raw
    ports:
      - "8080:8080"
```

### What OS are you seeing the problem on?

Other (please state in Anything else? section)

### ARM Release Version or Git hash (You can find this in the arm settings page)

d64f82f88880cb065b552919326fc4d13b18ea49

### Expected behavior?

I would expect it to stop attempting to eject after the first attempt.

### Relevant log output

```shell
From logs, 


Apr  9 15:30:59 af709d217333 ARM: eject: device name is `/dev/sr0'
Apr  9 16:00:26 af709d217333 ARM: eject: expanded name is `/dev/sr0'
Apr  9 16:00:26 af709d217333 ARM: eject: `/dev/sr0' is not mounted
Apr  9 16:00:26 af709d217333 ARM: eject: `/dev/sr0' is not a mount point
Apr  9 16:00:26 af709d217333 ARM: eject: `/dev/sr0' is not a multipartition device
Apr  9 16:00:26 af709d217333 ARM: eject: trying to eject `/dev/sr0' using SCSI commands
Apr  9 16:00:26 af709d217333 ARM: eject: SCSI eject succeeded
Apr  9 16:00:26 af709d217333 ARM: Entering docker wrapper
Apr  9 16:00:26 af709d217333 ARM: [ARM] Not CD, Blu-ray, DVD or Data. Bailing out on sr0
Apr  9 16:00:27 af709d217333 ARM: Entering docker wrapper
Apr  9 16:00:27 af709d217333 ARM: [ARM] Not CD, Blu-ray, DVD or Data. Bailing out on sr0
Apr  9 16:00:29 af709d217333 ARM: Entering docker wrapper
Apr  9 16:00:29 af709d217333 ARM: [ARM] Not CD, Blu-ray, DVD or Data. Bailing out on sr0
```
```


### Anything else? ( I am going to be editing this area continuously with observations)

❯ cat /etc/os-release       
PRETTY_NAME="Ubuntu 22.04.2 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.2 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy

When I unplug the drive from the host the messages stop. 

When I eject a disc using the button on the blue ray player, this symptom also shows itself. I need to confirm if the

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by TallPatrick** [comment]
I am having the same issue.  Also with ARM running inside a Proxmox virtual machine.  The model of drive I am using is a Pioneer Electronics BDR-XS07UHD 6x Slot Loading drive.  If there's anything I can do to provide a dump of the drive firmware or ARM logs, please let me know, I can reproduce this bug very consistently.

**Comment by BaelfireNightshd** [comment]
I also have this issue. Pioneer BDR-XD08UUMB-S. It doesn't even try to read the drive and in fact it seems that the drive gets stuck in eject mode until the ARM docker is restarted. It just ejects immediately. I know it's not trying to read the disk because it'll eject even with no disk in the drive.

**Comment by Tjkent88** [comment]
This is a duplicate of 
https://github.com/automatic-ripping-machine/automatic-ripping-machine/issues/779
closing in favor of that ticket.
